torch/nn/_intrinsic/qat/modules/conv_bn_relu.py

```diff
@@ -0,0 +1,227 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+import math
+import torch
+from torch.nn.modules.conv import _ConvNdBase
```

Comment:
from torch.nn.modules.conv import _ConvNdBase: Gives an import error, did you mean _ConvNd? Why not Conv2d?